WS2012R2/lisa/utilFunctions.ps1

```diff
@@ -253,6 +260,16 @@ function SetTestResult([String] $testName, [String] $completionCode)
             $newTestCase.failure.InnerText = "Test $testName Aborted."
         }
     }
+
+    $testID = GetTestID $testName $xmlData
+    foreach ($property in $newTestCase.properties.property)
+    {
+        if ($property.name -eq "TC_COVERED")
+        {
+            $property.value = $testID
```

Comment:
If the TC_COVERED field is missing from xml for a test case, this error is displayed by LISA at runtime:

Info :    GetTestID(Verify_LIS_Modules)
Cannot set "value" because only strings can be used as values to set XmlNode properties.
At F:\lis-test\WS2012R2\lisa\utilFunctions.ps1:269 char:13
+             $property.value = $testID
+             ~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], SetValueException
    + FullyQualifiedErrorId : XmlNodeSetShouldBeAString